STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -275,6 +275,32 @@ extended with a new `delegations` attribute:
 ~~~
 {: #fig-account-object title="Example Account object with delegations"}
 
+##### Delegation Lists
+
+Each account object includes a `delegations` URL from which a list of
+delegation configurations created by the IdO can be fetched via POST-as-GET
+request.  The result of the request MUST be a JSON object whose `delegations`
+field is an array of URLs, each identifying a delegation configuration made
+available to the NDC account ({{sec-delegation-objects}}).  The server MAY
+return an incomplete list, along with a Link header field with a `next` link
+relation indicating where further entries can be acquired.
+
+~~~
+HTTP/1.1 200 OK
+Content-Type: application/json
+Link: <https://acme.ido.example/acme/directory>;rel="index"
+Link: <https://acme.ido.example/acme/delegations/adFqoz?cursor=2>;rel="next"
+
```

Comment:
The outer map is unnecessary IMHO, but it is in line with RFC 8555, so fine.

Reply:
I stand corrected: https://softwareengineering.stackexchange.com/questions/286293/whats-the-best-way-to-return-an-array-as-a-response-in-a-restful-api